gobblin-service/src/main/java/org/apache/gobblin/service/modules/orchestration/DagProcessingEngine.java

```diff
@@ -149,6 +158,13 @@ public void run() {
           dagTask.conclude();
           log.info(dagProc.contextualizeStatus("concluded dagTask"));
         } catch (Exception e) {
+          if(!DagProcessingEngine.isTransientException(e)) {
+            log.error("Ignoring non transient exception. DagTask {} will conclude and will not be retried. Exception - {} ",
```

Comment:
don't see `toString()` for dagTask, what does dagTask log? It would be useful to have dagId & dagAction in the log.

Reply:
Thanks for pointing out, I had moved the existing log as it is, updated it now.